ols/app/models/config.py

```diff
@@ -24,7 +24,7 @@ def _get_attribute_from_file(data: dict, file_name_key: str) -> Optional[str]:
     """Retrieve value of an attribute from a file."""
     file_path = data.get(file_name_key)
     if file_path is not None:
-        with open(file_path, mode="r", encoding="utf-8") as f:
```

Comment:
IRC we added it there to be explicit. Right?

Reply:
probably yes, will redo this line :)

Reply:
aaaaand....gone @onmete :)